learning-objectives/data.yml

```diff
@@ -196,6 +196,61 @@ java:
     - persistance
     - querying
 
+python:
+  - data-types:
+    - primitive
+    - strings
+    - lists
+    - tuples
+    - dictionaries
+    - sets
+  - variables
+  - conditionals # ternary elif
```

Comment:
```suggestion
  - conditions
```

Reply:
ok mejor decir `conditions` que `conditionals`?

Reply:
not really, my bad